core/kazoo_documents/src/kazoo_documents.app.src

```diff
@@ -7,13 +7,12 @@
  ,{applications, [ kernel
                  , stdlib
                  , crypto
-
+                 , kazoo_convert
```

Comment:
@swysor i think you should add this to `fax` and `teletype`. i don't like it being part of `kazoo_documents`